lib/board.rb

```diff
@@ -0,0 +1,19 @@
+require_relative 'logic'
+# Board class to create a board
+class Board < Game
+  def display
+    print " #{@cells[0]} | #{@cells[1]} | #{@cells[2]} \n",
```

Comment:
Remove the print from board.rb and move into main.rb

Reply:
Removed the print like requested